extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -88,6 +88,10 @@
 	'user-identity-box-avatar-error-cantwrite' => 'Error: Could not write to temporary directory',
 	'user-identity-box-avatar-error-size' => 'Error: Avatar file is too big (max {{PLURAL:$1|$1|$1}}KB)',
 	'user-identity-box-avatar-error' => 'Error: There was internal error while saving avatar',
+	'user-identity-box-clear' => 'Clear profile contents',
+	'user-identity-box-clear-confirmation' => 'Are you sure you want to clear profile contents',
+	'user-identity-box-clear-success' => 'Successfully deleted user profile contents',
+	'user-identity-box-clear-notarget' => 'The specified user could not be found',
```

Comment:
Let's remember to push the updated i18n strings to Crowdin and file a ticket in LOC to translate them.